data/json/monsters.json

```diff
@@ -1808,6 +1808,105 @@
         "flags":["SEES", "STUMBLES", "POISON", "BLEED", "NO_BREATHE", "REVIVES", "REVIVES_HEALTHY", "NO_NECRO", "BONES"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_zombie_scorched_master",
+    "name": "zombie abomination",
```

Comment:
Abomination suggests something powerful physically.

Also, burned necro has the same name, which is not OK.